src/content/app-platform/getting-started/index.md

```diff
@@ -148,7 +148,7 @@ allows installing multiple instances of an app.
 Now lets check the app using the `kubectl gs get app` command.
 
 ```nohighlight
-kubectl -n ${CLUSTER} gs get app nginx-ingress-controller-app -o yaml
+kubectl gs -n ${CLUSTER} get app nginx-ingress-controller-app -o yaml
```

Comment:
Namespace flag needs to be after the gs plugin name.

I fixed the other commands but missed this one. Will go with a ping.